docs/program/SDK-as-client.md

```diff
@@ -1,44 +1,53 @@
 ---
-title: "Using Our SDKs for a Client Application"
```

Comment:
Definitely don't have to make this change, just a suggestion to make this more applicable from where in the docs we are linking to this page